docs/data-management-on-carbondata.md

```diff
@@ -126,20 +126,33 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 
    - **Local Dictionary Configuration**
      
-     Local Dictionary is generated only for no-dictionary string/varchar datatype columns. It helps in:
+     Local Dictionary is generated only for string/varchar datatype columns which are not included in dictionary include. It helps in:
      1. Getting more compression on dimension columns with less cardinality.
      2. Filter queries and full scan queries on No-dictionary columns with local dictionary will be faster as filter will be done on encoded data.
      3. Reducing the store size and memory footprint as only unique values will be stored as part of local dictionary and corresponding data will be stored as encoded data.
-   
-     By default, Local Dictionary will be enabled and generated for all no-dictionary string/varchar datatype columns.
+ 
+ **The cost for Local Dictionary:** The memory size will increase when local dictionary is enabled.
+ 
+ **NOTE:** Following Data Types are not Supported for Local Dictionary: 
+  * SMALLINT
+  * INTEGER
+  * BIGINT
+  * DOUBLE
+  * DECIMAL
+  * TIMESTAMP
+  * DATE
+  * CHAR
+  * BOOLEAN
+  
+     By default, Local Dictionary will be disabled.
           
      Users will be able to pass following properties in create table command: 
           
      | Properties | Default value | Description |
      | ---------- | ------------- | ----------- |
-     | LOCAL_DICTIONARY_ENABLE | false | By default, local dictionary will not be enabled for the table | 
-     | LOCAL_DICTIONARY_THRESHOLD | 10000 | The maximum cardinality for local dictionary generation (range- 1000 to 100000) |
-     | LOCAL_DICTIONARY_INCLUDE | all no-dictionary string/varchar columns | Columns for which Local Dictionary is generated. |
+     | LOCAL_DICTIONARY_ENABLE | false | By default, local dictionary will be disabled for the table |
+     | LOCAL_DICTIONARY_THRESHOLD | 10000 | The maximum cardinality for local dictionary generation (maximum - 100000) |
```

Comment:
description not correct. need to explain what threshold means and what happens beyond threshold